pentasharp/ViewModel/BusSchedul/EditScheduleViewModel.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using pentasharp.Models.Enums;
+
+namespace pentasharp.ViewModel.BusSchedul
+{
+    /// <summary>
+    /// Represents the data required to edit an existing bus schedule.
+    /// </summary>
+    public class EditScheduleViewModel
+    {
+        /// <summary>
+        /// Unique identifier for the route associated with the schedule.
+        /// </summary>
+        [Required]
```

Comment:
remove data annotation since we decided to stop using it, and you already have models without them in this PR